tests/python/pants_test/backend/jvm/tasks/jvm_compile/rsc/test_rsc_compile_integration.py

```diff
@@ -96,3 +96,21 @@ def test_executing_multi_target_binary_hermetic(self):
           workdir, config)
         self.assert_success(pants_run)
         self.assertIn('Hello, Resource World!', pants_run.stdout_data)
+
+  def test_java_with_transitive_exported_scala_dep(self):
+    with temporary_dir() as cache_dir:
```

Comment:
while working on this test, realized this is rewritten in every test - abstract out? (of course, I could do this too :))

Reply:
Hm. That might make some sense. Maybe a helper fn that constructs a default config and applies modifications? I'm not sure if it'd buy enough though.